ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts

```diff
@@ -0,0 +1,37 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: 0x3B8, // @TODO: change to zoneid entry when available
```

Comment:
This should already exist!

Reply:
My local tree was pre-EW-update when I wrote these originally.